### ISSUE TITLE
Default dashboard view is in the middle of metrics

### ISSUE THREAD

**Issue by adonahue** [rifflearning/zenhub-platform-project#65]
After a meeting, or when you click on a meeting from the timeline, the default view of the metrics is in the middle of the page.

### Steps to Reproduce

1. Go to My Riffs in staging. https://staging.riffplatform.com/riffs
2. Click on a meeting from the timeline
3.  See error

### Expected behavior

 The expected behavior is that it be at the top of the page, as it does in Beta https://beta.riffplatform.com/riffs. Where it shows the meeting summary, the speaking time, and influence metrics. 

### Screenshots
![image.png](https://images.zenhubusercontent.com/5cbf1c836caba0089f532d44/e495d7bb-94d7-4081-8185-52e6e0c11e97)

On @mlippert 's screen it isn't as obvious but now that you point it out see screenshot below.
![image.png](https://images.zenhubusercontent.com/5cb73c174005bf783cb0eaf4/600a82fc-70d5-4bb2-a4dc-15a69e46d839)

### Desktop (please complete the following information):
 - OS: Windows 10
 - Browser: chrome, FF

### Possible Cause
The metrics now each show a spinner until their data is available. I think that the size of the chart when displaying a spinner is smaller than when it shows the visualization, and that when the chart is drawn the bottom of the window remains in view and the chart will be off the top of the window if it is taller than the spinner.

A different possible cause, and the one I now feel is more likely, is the a11y changes to the dashboard which set the focus. What I observed was that the info button on the timeline chart card was given focus bringing it into view. If that's required for a11y I'm not sure how to fix this. I'm not sure the way we programmatically set the focus is really a good UX though.

So the question I (@mlippert) have is why do we want to set the focus like this?
https://github.com/rifflearning/riff-rtc/blob/develop/src/components/Dashboard/Metrics/ChartCard.jsx#L79-L83
```
        // We want to have the focus on the info button when the chart card is
        // rendered w/ the info closed.
        if (!this.state.isInfoOpen) {
            this._infoBtn.current.focus();
}
```

**Comment by adonahue** [comment]
@arizzitano - can you please take a look at this and see if it's related to the a11y changes, and if so, fix. thx!

**Comment by arizzitano** [comment]
The desired functionality here is that focus shifts to the info button *only* after the info box is closed, not every time the component renders. This was a recommendation from the initial audit introduced in https://github.com/rifflearning/riff-rtc/pull/57/files and worked as expected when I audited it in April.

This regression is not related to the recent meeting mediator changes. It looks like the component has undergone some refactoring recently, which may have changed the focus shifting behavior.

**Comment by mlippert** [comment]
Well that certainly makes more sense @arizzitano . I knew it had nothing to do with the mediator changes. I'll see about making it work like you intended.

**Comment by adonahue** [comment]
The desired behavior is to replicate the pre- re-factor functionality, which can be observed on Beta. 

Tested on staging / Chrome 

** Expected Behavior ** 
- [ ] On hover - for both the meeting date and camera icon - in the navigation list should display the "pointed finger" cursor, indicating that they are a clickable item. In Chrome it it does not, FF is ok. 
- [ ] When a meeting is clicked in the navigation, the page should display from the top, so the metrics are at the top of the page. In chrome and FF the metrics reload, but the page view does not reset to the top. For very long navigation lists, the user can not see that the metrics have reloaded at all. 
- [ ] In chrome, a blue outline appears around the clickable navigation item. No outline should display. 
![image.png](https://images.zenhubusercontent.com/5cbf1c836caba0089f532d44/b1492e2a-a7f2-4c10-b0a2-f42c692a2946)

**Comment by adonahue** [comment]
@mlippert - moving this back to the backlog for you.

**Comment by adonahue** [comment]
@mlippert - it doesn't look like the issue of not jumping to the top of the page has been fixed. What prompted the move to the QA pipeline? Thanks!

**Comment by mlippert** [comment]
This fixed the problem of them jumping to the middle of the page, but it doesn't force jumping to the top of the page when a new meeting is selected.

**Comment by adonahue** [comment]
@mlippert - the desired functionality is when a link is clicked, for the page to display at the top, so that the user can see their metrics for the meeting they selected. Are there implementation problems with jumping to the top that should be part of how we consider this work? Thanks!

**Comment by mlippert** [comment]
This was part of the refactor and since the previous code didn't do anything to jump to the top of the page, the refactor isn't either. I was explicitly trying to not add code that did things differently other than cleanup. It seems there may have been some bugs that on chrome moved to the top of the page, but firefox never has, so I consider this a new feature, not a fix to the refactor work.

I can figure out how to do this, but it does feel like a new interruption story rather than continuation of an existing story.

**Comment by adonahue** [comment]
Yeah @mlippert , I see that as a totally valid point of view. Not the outcome I wanted, but makes sense in the context of the refactor. Thanks. With that in mind, marking this as Accepted.
